src/client/components/home/Home.js

```diff
@@ -1,28 +1,89 @@
 import { makeStyles } from '@material-ui/core';
 import Typography from '@material-ui/core/Typography';
-import React, { useContext } from 'react';
+import React, { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { APP_AUTHOR, APP_DESCRIPTION, APP_NAME } from '../../config/constants';
 import CollectionsGrid from '../collection/CollectionsGrid';
-import { CollectionContext } from '../CollectionProvider';
 import Seo from '../common/Seo';
+import { useCollections } from '../../utils/collections';
+import Loader from '../common/Loader';
+import Search from './Search';
 
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles((theme) => ({
   wrapper: {
     padding: '4vw',
   },
+  root: {
+    padding: theme.spacing(0.25, 0.5),
+    margin: '100px 0px',
+    display: 'flex',
+    alignItems: 'center',
+    width: '100%',
+  },
+  input: {
+    marginLeft: '20px',
+    flex: 6,
+  },
+  iconButton: {
+    padding: 10,
+  },
+  divider: {
+    height: 28,
+    margin: 4,
+  },
+  typographyMargin: {
+    margin: '10px 0px',
+  },
 }));
 
 function Home() {
   const { t } = useTranslation();
-  const { collections } = useContext(CollectionContext);
+  const [queried, setQueried] = useState(false);
```

Comment:
Wouldn't is be possible to check for `null` (versus `[]` for empty result)?

Reply:
Sorry @pyphilia  but I didn't understand your question. and where exactly?

Reply:
Sorry, the comment was considering line 83 with `{isLoading ? <Loader /> : queried ? renderResults() : null}`
Where I think you can simplify with `{isLoading ? <Loader /> :  renderResults()}`
And `renderResults()` would check for `null` (no search was performed), `[]` (no collection matching the results) and `[...]` rendering some results. That why `queried` is not necessary.